help/access/databases.md

```diff
@@ -1,282 +1,290 @@
----
-title: "Databases"
----
+# Databases
 
-## Connecting to a database
+```mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+```
 
-> Key concept: [_Entity_](../datagrok/objects.md)
->
->Certain classes of objects in Datagrok have a standard set of operations that can apply to them. We call these
-> objects _entities_. You can share _entities_, set permissions, annotate, reuse them in multiple ways, and more.
-> Connections, queries, tables, and table columns are all Datagrok _entities_.
->
->Key concept: [_Function_](../datagrok/functions/functions.md)
->
->Any action executed within Datagrok is a _function_. For example, a data query, a calculation, or sending an email are
-> all _functions_. _Functions_ are scriptable and auditable, can be parameterized, used in automation workflows, and
-> more.
+Datagrok has native support for relational databases ([30+ databases supported](supported-connectors.md)), allowing to do the following:
 
-### Database connection
+* Browse databases schemas, tables, and columns.
+* Query databases using parameterized queries.
+* Use query as a dynamically refreshed data source for dashboards.
+* Share connections and queries with others.
+* Configure user privileges.
+* Create data pipelines.
 
-A database connection lets you work with databases directly in Datagrok. You can connect to a supported database or a
-relational database that uses a Java Database Connectivity (JDBC) driver.
+:::note Developers
 
-Before you connect a database, the following prerequisites must be met:
+You can [create custom connectors](create-custom-connectors.md). You can also [connect to a data source, query data, and share connections programmatically](../develop/how-to/access-data.md/#connections).
 
-* Datagrok has a connector to the data source. Out-of-the-box, Datagrok connects to all major databases. For the
-  complete list of supported connectors, see [Connectors](supported-connectors.md).
-  > Developers: You can create [custom connectors](https://github.com/datagrok-ai/public/tree/master/connectors).
-  <!--need to create a How-to under Develop-->
-* You must have the appropriate privileges to the database. If you don't have database privileges, contact the database
-  administrator.
-* You must be able to authenticate the connection (typically, with a username and password).
+:::
 
-#### Add new connection
+## Connecting to database
+
+### Adding connection
+
+![Add new database connection](database-connection.gif)
```

Comment:
The gif is a bit confusing, probably should go straight to establishing the connection